src/Tools/dotnet-monitor/Dumps/DumpFilePathTemplate.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using System;
+using System.Globalization;
+using System.Net;
+using System.Text;
+using Utils = Microsoft.Diagnostics.Monitoring.WebApi.Utilities;
+
+namespace Microsoft.Diagnostics.Tools.Monitor
+{
+    internal sealed class DumpFilePathTemplate
+    {
+        private const char PercentSpecifier = '%';
+        private const char ProcessIdSpecifier = 'p';
+        private const char ExecutableSpecifier = 'e';
+        private const char HostSpecifier = 'h';
+        private const char TimeSpecifier = 't';
+        private const char DateTimeSpecifier = 'd';
```

Comment:
It looks like I shouldn't use `%d` for this as documentation also seems to allude to `%d` being acceptable for PID as well.